wellcomeml/ml/bert_semantic_equivalence.py

```diff
@@ -240,19 +242,25 @@ class SemanticEquivalenceMetaClassifier(SemanticEquivalenceClassifier):
     and model initialisation. Fit, predict, score remain intact
     """
 
-    def __init__(self, n_numerical_features, dropout_rate=0.1, **kwargs):
+    def __init__(self, n_numerical_features, dropout=False,
+                 batch_norm=False, dropout_rate=0.1, **kwargs):
```

Comment:
maybe you want to default to dropout 0 to maintain backwards compatibility

Reply:
That's what `dropout=False` does

Reply:
maybe you do not need two parameters then :P dropout 0 means no dropout is applied

Reply:
You are right. Maybe a merged too quick. In the next PR I will patch this change in.